homework-1/task-2/main.js

```diff
@@ -0,0 +1,85 @@
+const createВominoesBtn = document.querySelector('.domino-btn');
+const topRow = document.querySelector('.domino-up');
+const bottomRow = document.querySelector('.domino-down');
+const compareBtn = document.querySelector('.compare-domino');
+const showAnswer = document.querySelector('.show-answer');
+let negativAnswer;
+
+
+function topRandomDomino () {
+    let topDominoArray = [];
+    for(let i = 0; i < 6; i++) {
+        topDominoArray.push(Math.floor(Math.random() * (6 - 1) + 1));
+    }
+    return topDominoArray;
+}
+
+function bottomRandomDomino () {
```

Comment:
this two methods are equal, so you don't need to dublicat it , just create only one helper

Reply:
done!